examples/Go2Target.ipynb

```diff
@@ -0,0 +1,388 @@
+    "# ATPtg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the Pointing Component is not enabled, run the next cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await asyncio.sleep(5.)\n",
+    "print(\"Done\")\n",
+    "await atptg.tel_timeAndDate.next(flush=True, timeout=5)\n",
+    "await salobj.set_summary_state(atptg, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Going to the Target"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next cell sets the observatory location. This is needed to compute the Az/El of the target to set the camera rotation angle. We are trying to keep the angle close to zero. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "location = EarthLocation.from_geodetic(lon=-70.747698*u.deg,\n",
+    "                                       lat=-30.244728*u.deg,\n",
+    "                                       height=2663.0*u.m)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This next cell defines a target."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ra = Angle(\"20:25:38.85705\", unit=u.hour)\n",
+    "dec = Angle(\"-56:44:06.3230\", unit=u.deg)\n",
+    "target_name=\"Alf PAv\"\n",
+    "radec = ICRS(ra, dec)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This next cell will slew to the target and set the camera rotation angle to zero. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Figure out what is the rotPA that sets nasmith rotator close to zero.\n",
+    "time_data = await atptg.tel_timeAndDate.next(flush=True, timeout=2)\n",
+    "curr_time_atptg = Time(time_data.tai, format=\"mjd\", scale=\"tai\")\n",
+    "print(curr_time_atptg)\n",
+    "coord_frame_altaz = AltAz(location=location, obstime=curr_time_atptg)\n",
+    "alt_az = radec.transform_to(coord_frame_altaz)\n",
+    "\n",
+    "print(\"slew...\")\n",
+    "# await atmcs.cmd_startTracking.start(timeout=10)\n",
+    "await atptg.cmd_raDecTarget.set_start(\n",
+    "    targetName=target_name,\n",
+    "    targetInstance=ATPtg.TargetInstances.CURRENT,\n",
+    "    frame=ATPtg.CoordFrame.ICRS,\n",
+    "    epoch=2000,  # should be ignored: no parallax or proper motion\n",
+    "    equinox=2000,  # should be ignored for ICRS\n",
+    "    ra=radec.ra.hour,\n",
+    "    declination=radec.dec.deg,\n",
+    "    parallax=0,\n",
+    "    pmRA=0,\n",
+    "    pmDec=0,\n",
+    "    rv=0,\n",
+    "    dRA=0,\n",
+    "    dDec=0,\n",
+    "    rotPA=180.-alt_az.alt.deg,\n",
```

Comment:
Say this is where the rotator position is optimized to be near zero. Also say what would be the default value if you weren't trying to set it near zero